packages/connect-popup/src/view/common.tsx

```diff
@@ -152,6 +150,10 @@ export const initMessageChannel = async (
         }
     }
 
+    if (!iframe) {
+        throw ERRORS.TypedError('Popup_ConnectionMissing');
+    }
+
```

Comment:
as stated in my comment before  - this might be fixing the problem becuase code exectuion never gets here in certain cases - it maybe stops on line 142? but iframe reference is still not set maybe?